test/integration/sample30-default-order-by.ts

```diff
@@ -0,0 +1,97 @@
+import "reflect-metadata";
```

Comment:
Im very sorry but can you please isolate your test from samples. What you see in "integration" is old way of making tests. New way of making tests is in `functional` directory. Can you please move your test there and isolate it from sample? (e.g. create entities right in the test directory), just take a look how other tests in `functional` directory are organised. They also are testing all connection types, not only postgres